roscala/src/test/scala/coop/rchain/rosette/prim/TupleSpec.scala

```diff
@@ -78,4 +78,52 @@ class TupleSpec extends FlatSpec with Matchers {
     tplRcons.fn(newCtxt) should be('left)
   }
 
+  /** tuple-concat */
+  "tplConcat" should "correctly concat n Tuples" in {
+    val tup1 = Seq(RFixnum(1), RFixnum(2))
+    val tup2 = Seq(RFixnum(3))
+    val tup3 = Seq(RFixnum(4), RFixnum(5), RFixnum(6))
+    val tups = Seq(Tuple(tup1), Tuple(tup2), Tuple(tup3))
+
+    val newCtxt =
+      ctxt.copy(nargs = 3, argvec = Tuple(tups))
+
+    tplConcat.fn(newCtxt) should be(
+      Right(
+        Tuple.rcons(Tuple.rcons(Tuple.rcons(Tuple.rcons(Tuple(tup1),
+                                                        RFixnum(3)),
```

Comment:
Something is wrong with indentation.

Reply:
The automatic formatting done in sbt keeps changing my indentation.  I gave up fighting it.

Reply:
Scalafmt is not perfect but better than nothing. I'll leave it to @DoSOfRedRiver to decide if it's fine in this case or not.

Reply:
98 column length

Reply:
@kjmcqueen do a `sbt refresh` and see if it changes since we altered the scalafmt configuration to allow for longer lines.

Reply:
sbt reload didn't seem to make any difference.

If we are going to use Scalafmt to enforce formatting, then we should not be holding up PRs for formatting. Fix the Scalafmt formatting in a separate PR instead.

Comments like "Something is wrong with indentation." are not productive in this environment.

Reply:
No, this is wrong. Pull requests MUST correctly implement the project's style guide. You are only allowed to merge stylish code. Run scalafmt and commit the change BEFORE you create your pull request.

Reply:
I did.  That is what he's complaining about.

Reply:
I'll have a different approach shortly.